backend/src/main/java/com/graccasoft/schoolinvoicing/controller/InvoiceController.java

```diff
@@ -2,10 +2,8 @@
 
 import com.graccasoft.schoolinvoicing.dto.GenericResponse;
 import com.graccasoft.schoolinvoicing.service.InvoiceService;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("invoice")
```

Comment:
```suggestion
@RequestMapping("invoices")
```
Best practise to use plural for nouns in the REST urls